onnxruntime/core/graph/model.cc

```diff
@@ -828,6 +828,14 @@ common::Status Model::LoadFromOrtFormat(const fbs::Model& fbs_model,
   ORT_RETURN_IF(nullptr == fbs_graph, "Graph is null. Invalid ORT format model.");
 
 #if !defined(ORT_MINIMAL_BUILD)
+  // add the opset imports to the model_proto in case we're updating an ORT format model and need those to be
+  // included when SaveToOrtFormat is called later
+  for (const auto& domain : domain_to_version) {
```

Comment:
nit: could use structured bindings to get better names than `first` and `second`
```suggestion
  for (const auto& [domain, version] : domain_to_version) {
```